LondonBritishLibrary/orient/BLorient11718A.xml

```diff
@@ -0,0 +1,226 @@
+                                <support>
+                                    <material key="paper"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">79</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>325</height>
+                                        <width>215</width>
+                                    </dimensions>
+                                </extent> 
+                                <foliation>The volume has no unified pagination, but each codicological unit has its own folio or page numbers.</foliation>
+                            </supportDesc>  
+                        </objectDesc>
+                        <bindingDesc>
+                            <binding xml:id="binding" contemporary="false">
+                                <decoNote xml:id="b1" type="Boards">European binding provided by the <placeName ref="INS00001BL"/>.</decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin><origDate notBefore="1923" notAfter="1924">1923/1924</origDate></origin>
+                        <provenance></provenance>
+                        <acquisition>On the verso of the last leaf - a note: 
+                            'Presented by <persName ref="PRS14446Douglas">Canon John A. Douglas</persName>. 
+                            <date when="1941">Jan. 1941</date>'</acquisition>
+                    </history>
+                    <additional>
+                        <adminInfo>
+                            <recordHist>
+                                <source>
+                                    <listBibl type="catalogue">
+                                        <bibl>
+                                            <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                            <citedRange unit="page">81</citedRange>
+                                        </bibl> 
+                                    </listBibl>
+                                </source>
+                            </recordHist>
+                        </adminInfo>
+                    </additional>
+                    <msPart xml:id="p1">
+                        <msIdentifier/>
+                        <msContents>
+                            <summary/>
+                            <msItem xml:id="p1_i1">
+                                <locus from="2" to="82"/>
+                                <title>Answers given by Ethiopian Doctors to the questions addressed by Bishop Gwynne of Khartoum</title>
+                                <msItem xml:id="p1_i1.1">
+                                    <locus from="2" to="8"/>
+                                    <title>List of sixty questions</title>
```

Comment:
No ID for this text?

Reply:
I understood, that this is a letter from the community of scholars of the Ethiopian Church around Ras Tafari to Bishop Gwynn in Khartoum. I did not expect any circulation inside Ethiopia for it and did not investigate much until now.